core/src/main/java/com/datastax/oss/driver/api/core/metadata/schema/RelationMetadata.java

```diff
@@ -30,6 +31,22 @@
   /** The unique id generated by the server for this element. */
   UUID getId();
 
+  /**
+   * Convenience method to get all the primary key columns (partition key + clustering columns) in a
+   * single call.
+   *
+   * <p>Note that this creates a new list instance on each call.
+   *
+   * @see #getPartitionKey()
+   * @see #getClusteringColumns()
+   */
+  default List<ColumnMetadata> getPrimaryKey() {
+    return ImmutableList.<ColumnMetadata>builder()
+        .addAll(getPartitionKey())
+        .addAll(getClusteringColumns().keySet())
```

Comment:
Obviously this doesn't return the clustering order. The 3.x version didn't either, so I guess it's not a problem.